src/core/meshcop/dtls.cpp

```diff
@@ -68,15 +68,47 @@ Dtls::Dtls(Instance &aInstance)
     , mSendHandler(NULL)
     , mContext(NULL)
     , mClient(false)
+    , mApplicationCoapSecure(false)
```

Comment:
Is it possible to avoid application layer flag in this module?

Reply:
yes. now check by cipher suite.